web/components/elements/ErrorPane.module.scss

```diff
@@ -1,4 +1,8 @@
 .errorPane {
     max-width: 480px;
     color: #FF4B55;
+}
+.warningPane {
+    max-width: 480px;
+    color: #CCAA00;
```

Comment:
This colour isn't great for contrast, have you consulted the Figma Oracles?

https://www.figma.com/file/KhVjYAzy4oX3eIqwu9KQxN/Element-library?node-id=4779%3A18811
https://www.figma.com/file/X4XTH9iS2KGJ2wFKDqkyed/Compound?node-id=557%3A0